provision/ubuntu/install.sh

```diff
@@ -156,6 +156,10 @@ nameserver 8.8.8.8
 nameserver 8.8.4.4
 EOF
 
+# CoreDumps https://github.com/cilium/cilium/issues/3399
+sudo systemctl disable apport.service
+sudo sysctl -w kernel.core_pattern=/tmp/core.%e.%p.%t
```

Comment:
@eloycoto This is not enough, the setting disappears on reboot. Add this right after:
```
sudo sh -c 'sysctl kernel.core_pattern > /etc/sysctl.d/66-core-pattern.conf'
```